src/components/SearchFacet/SearchFacetModal/SearchFacetModal.tsx

```diff
@@ -192,7 +191,9 @@ const FacetDownloadButton = () => {
     enabled,
   });
 
-  const formatData = useCallback(() => pipe(pluck('val'), join('\n'))(treeData), [treeData]);
+  const pluckVal = (list: FacetItem[]) => pluck('val', list);
+  const cleanup = replace(/^0\//, ''); // remove prefix
+  const formatData = useCallback(() => pipe(pluckVal, map<string, string>(cleanup), join('\n'))(treeData), [treeData]);
```

Comment:
```suggestion
  const formatData = useCallback(
    () => pipe<[FacetItem[]], string[], string[], string>(pluck('val'), map(replace(/^0\//, '')), join('\n'))(treeData),
    [treeData],
  );
```

Cleans this up some.  also `pluck` is automatically curried so you don't have to wrap it